### PR TITLE
added test-sheet in skipped streams

### PR DESCRIPTION
**Pull request by namrata270998**
# Description of change
- Added test-sheet to skip for integration tests which is added in another branch's test

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tap_google_sheets/test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import dateutil.parser
```
Comment (kspeer825) [inline tap_google_sheets/test.py:1]: What is the purpose of this file? I don't think it should be included.

**Review by kspeer825 — CHANGES_REQUESTED** [review]

File: tap_google_sheets/test.py (hunk 1 above)
Comment (namrata270998) [reply tap_google_sheets/test.py:1]: To use the `dateutil.parser.parse()` on [L10](https://github.com/singer-io/tap-google-sheets/pull/79/files#diff-0594dc805436cc5bf1a25bfa3d3e597e3daf988d3ccde3c497c6ad4953a56b97R10) in the function `strptime_to_utc`, we have imported the dateutil.parser.

Comment (kspeer825) [reply tap_google_sheets/test.py:1]: Why is the `test.py` file included in this PR?

**Review by kspeer825 — CHANGES_REQUESTED** [review]

File: tap_google_sheets/test.py (hunk 1 above)
Comment (namrata270998) [reply tap_google_sheets/test.py]: My bad, it was done by mistake. Removed the file.

**Review by kspeer825 — APPROVED** [review]

**Pull request merged**